cockatrice/src/carddatabase.h

```diff
@@ -78,7 +78,12 @@ class CardInfo : public QObject {
     QString powtough;
     QString text;
     QStringList colors;
+    // the cards i'm related to
     QStringList relatedCards;
+    // the card i'm reverse-related to
+    QStringList reverseRelatedCards;
+    // the cards thare are reverse-related to me
+    QStringList reverseRelatedCardsToMe;
```

Comment:
We're both reading from and writing to the xml file the contents of the ```relatedCards``` and ```reverseRelatedCards``` fields. If we merge the fields, the information will be duplicated into the xml at the first card database save.